packages/circuit-ui/cli/migrate/button-default-size.ts

```diff
@@ -38,10 +38,16 @@ function transformFactory(
             type: 'JSXIdentifier',
             name: 'size',
           },
+          // These TypeScript errors popped up when bumping @types/jscodeshift
+          // to 0.11.5. The codemod works.
+          /* eslint-disable @typescript-eslint/ban-ts-comment */
           value: {
+            // @ts-ignore
             type,
+            // @ts-ignore
             value: 'mega',
           },
+          /* eslint-enable @typescript-eslint/ban-ts-comment */
```

Comment:
The ESLint rule is right here, `@ts-expect-error` should be used in stead of `@ts-ignore`.

Reply:
Why?

I'd use `@ts-expect-error` if the error is expected. Here it's not, it rather looks like a bug with jscodeshift types (since the codemods obviously work)

What am I missing?

Reply:
`@ts-ignore` turns off type-checking completely. `@ts-expect-error` tells the compiler that we're okay with the error **and** that it should let us know if there is no error. In other words it'll let you know once the error has been resolved and the workaround can be removed.

Reply:
Sure—but since we're not planning on touching these files anyways, I don't think it's very important in this case. Feel free to change if you feel strongly about it of course! 🙂 

I'll clean up all old codemods from the repo to avoid causing too many TS issues as discussed—this [wasn't the first time](https://github.com/sumup-oss/circuit-ui/pull/1643/files#diff-118b0c24392ceeeafc825b2779b2aa697a7d59ad77e3ecd72ae8ef3341bf8387L56-L58) and I don't want to spend too much time on this

Reply:
I don't feel that strongly — this was more out of principle and in case you weren't familiar with `@ts-expect-error`.